Content.Server/Traitor/Components/AutoTraitorComponent.cs

```diff
@@ -19,4 +19,10 @@ public sealed partial class AutoTraitorComponent : Component
     /// </summary>
     [DataField("giveObjectives"), ViewVariables(VVAccess.ReadWrite)]
     public bool GiveObjectives = true;
+
+    /// <summary>
+    /// the probability of the target becoming the antagonist
+    /// </summary>
+    [DataField]
+    public float Prob = 1f;
```

Comment:
This should be handled higher up, autotraitor is supposed to always traitor them and this will just multiply with whatever gamerule chances there are.